io/disk/dbench.py.data/dbench.yaml

```diff
@@ -1,3 +1,4 @@
+disk:
 setup:
     duration: !mux
         default:
```

Comment:
@smruti77 you need to remove these lines as well.
And make sure Readme is also updated, if needed, in all files.